### PR TITLE
add windowId as a parameter for MouseWheel

### PR DESCRIPTION
**Pull request by timlyeee**
Re: #

### Changelog

* windowId = 1 by default
* add window id as a parameter for mouse wheel event

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Review by PatriceJiang — APPROVED** [review]

**Review by PPpro — APPROVED** [review]

File: cocos/input/types/event/event-mouse.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ export class EventMouse extends Event {
      * @en The unique ID of SystemWindow, which triggerd the event
      * @zh 触发此事件的系统窗口 ID
      */
-    public windowId = 0;
+    public windowId = 1;
```
Comment (minggo) [inline cocos/input/types/event/event-mouse.ts:106]: Why change default value to 1?

Comment (timlyeee) [reply cocos/input/types/event/event-mouse.ts:106]: It should be 1 as native.

Comment (minggo) [reply cocos/input/types/event/event-mouse.ts:106]: 0 is invalid id, but default value use it is ok as it should be assigned to a valid id.

**Comment by timlyeee** [comment]
企业微信截图_16689959896079
The initialization of window id shows on ISystemWindow.h

**Review by minggo — APPROVED** [review]

**Pull request merged**